src/System.Private.CoreLib/shared/System/Text/ASCIIEncoding.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
Is `#nullable` a new / [undocumented](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives/) directive?

Reply:
it's a new experimental feature. we are currently adding it to system.private.corelib to check if it works good in practice before we recommend everyone to use (besides kinda pointless to use it if framework stuff is not annotated yet).

Reply:
https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references